src/Cake.Curl/CurlSettings.cs

```diff
@@ -35,5 +36,10 @@ public class CurlSettings : ToolSettings
         /// Gets or sets the password to use when authenticating to the remote host.
         /// </summary>
         public string Password { get; set; }
+
+        /// <summary>
+        /// Gets or sets the header to use when communicating to the remote host.
+        /// </summary>
+        public Dictionary<string, string> Headers { get; set; }
```

Comment:
Whenever possible, I prefer abstractions over concrete types. In this case, I would rather have the `Headers` property be of type `IDictionary<string, string>`. This would also be more consistent with the rest of the Cake API (for example, see the [`ToolSettings.EnvironmentVariables`](http://cakebuild.net/api/Cake.Core.Tooling/ToolSettings/ACBC8906) property).